code/src/App.js

```diff
@@ -1,9 +1,24 @@
-import React from 'react'
+import React from "react";
+import { Switch, Route } from "react-router-dom";
 
-export const App = () => {
+import Header from "./components/header/header.component";
+import Survey from "./components/survey/survey.component";
+import Main from "./pages/main/main.component";
+import NotFound from "./pages/not-found/not-found.component";
+import Footer from "./components/footer/footer.component";
+
+function App() {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
```

Comment:
Maybe take this away part away ☺️?

Reply:
Oh! I forgot it! Thanks to mention it. :pray: